.github/ISSUE_TEMPLATE/where-to-report-issues.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "⚠️ Where to report issues ?"
+about: File issues on the main Eclipse Che repository at https://github.com/eclipse/che/issues
```

Comment:
```suggestion
about: File issues in the main Eclipse Che repository at [https://github.com/eclipse/che/issues](https://github.com/eclipse/che/issues).
```

Reply:
Here I can't merge the link syntax as the about section is not written in markdown so user will see ugly stuff
I've fixed my error related to `on`/`in`

Reply:
you can see how it's rendered on https://github.com/che-incubator/chectl/issues/new/choose